<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите время мероприятия:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_date">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>85</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите дату мероприятия:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>20</number>
    </property>
    <property name="text">
     <string>Введите название мероприятия:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_geo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>195</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить запись</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="input_date">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>105</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>23</hour>
      <minute>58</minute>
      <second>30</second>
      <year>2025</year>
      <month>1</month>
      <day>23</day>
     </datetime>
    </property>
   </widget>
   <widget class="QLabel" name="label_geo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>175</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите место мероприятия:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_add">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ДОБАВИТЬ ЗАПИСЬ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_replace">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ИЗМЕНИТЬ ЗАПИСЬ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_delete">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>УДАЛИТЬ ЗАПИСЬ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_text_replace">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите запись, которую хотите изменить:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="input_replace_box">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_replace_name">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>85</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить название</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_replace_date">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить дату</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_replace_name">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>105</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_replace_geo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_replace_geo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить место мероприятия</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_replace_time">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>175</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить время</string>
    </property>
   </widget>
   <widget class="QPushButton" name="replaceNameButton">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>104</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>✔</string>
    </property>
   </widget>
   <widget class="QPushButton" name="replaceDateButton">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>149</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>✔</string>
    </property>
   </widget>
   <widget class="QPushButton" name="replaceTimeButton">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>194</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>✔</string>
    </property>
   </widget>
   <widget class="QPushButton" name="replaceGeoButton">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>239</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>✔</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_text_delete">
    <property name="geometry">
     <rect>
      <x>511</x>
      <y>40</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите запись, которую хотите удалить:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="input_delete_name">
    <property name="geometry">
     <rect>
      <x>511</x>
      <y>60</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>105</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="input_time">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>23</hour>
      <minute>58</minute>
      <second>30</second>
      <year>2000</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ОШИБКА: Заполните все поля</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="input_replace_date">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>23</hour>
      <minute>58</minute>
      <second>30</second>
      <year>2025</year>
      <month>1</month>
      <day>23</day>
     </datetime>
    </property>
   </widget>
   <widget class="QTimeEdit" name="input_replace_time">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>195</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>23</hour>
      <minute>58</minute>
      <second>30</second>
      <year>2025</year>
      <month>1</month>
      <day>23</day>
     </datetime>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
